docs/intro/install.rst

```diff
@@ -127,16 +127,16 @@ To install Scrapy on Windows using ``pip`` :
 
 #. Download and execute `Microsoft C++ Build Tools`_ to install the Visual Studio Installer.
 
-#. Open Visual Studio Installer, install Visual Studio and modify installation.
+#. Open Visual Studio Installer, install `Visual Studio`_.
```

Comment:
This sentence still sounds like you first install Visual Studio, and then you can access a “Workloads” section. But from reading [the upstream documentation](https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019) it sounds like you get that Workloads section as soon as you open the installer. Isn’t that so?

Reply:
Sorry I forgot to change that and also I figured out that I was a little bit wrong [here](https://github.com/scrapy/scrapy/pull/4736#discussion_r475101391). **Desktop development with C++** comes when we install [VS IDE](https://visualstudio.microsoft.com/downloads) too along with other things but here we need only [build tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/) so [reddit post](https://www.reddit.com/r/scrapy/comments/hzgtwt/how_to_install_scrapy_using_pip_for_python3_in/) was right, this should be **C++ build tools**